autotest/test_swf_dfw_gwf.py

```diff
@@ -58,20 +58,20 @@ def build_models(idx, test):
     add_swf_model(sim)
     add_gwf_model(sim)
 
-    swfgwf_data = [
+    gwfswf_data = [
```

Comment:
Is there are a reason for changing this?  My thinking is that the name should be swf-gwf to loosely support the idea of upper to lower.  This order also corresponds to the cellids in the swfgwf_data list.  The first tuple there is a SWF reach number and the second is the GWF cellid.

Reply:
Understood. I had two reasons for doing this: consistency with GWT, GWE (and PRT), and it makes [the logic in `src/meson.build`](https://github.com/MODFLOW-USGS/modflow6/pull/1532/files#diff-f28598af2e23aa5d2bc7c72e022ae2c56a33802eb970afffaeca1e40607f97feR27) to programmatically define exchanges slightly simpler if we can assume the order &mdash; but can just check the `src/Exchange` directory instead to see which exchanges are present. I will restore the original swf-gwf namings.